core/src/main/java/com/sequenceiq/cloudbreak/repository/StackRepository.java

```diff
@@ -103,7 +103,7 @@
             + "FROM Stack s LEFT JOIN s.cluster c LEFT JOIN s.workspace WHERE s.terminated = null AND (s.type is not 'TEMPLATE' OR s.type is null)")
     List<StackTtlView> findAllAlive();
 
-    @Query("SELECT s FROM Stack s LEFT JOIN FETCH s.instanceGroups ig "
+    @Query("SELECT s FROM Stack s LEFT JOIN FETCH s.instanceGroups ig LEFT JOIN FETCH ig.instanceMetaData im "
```

Comment:
Is it possible to reduce the size of this resultset? This might get huge and we are only using the InstanceGroups from it.

Reply:
good catch, we only need the image so I will modify the query

Reply:
modified both cb and freeipa queries to just return the images instead of the full stack objects